packages/hoppscotch-common/src/helpers/functional/json.ts

```diff
@@ -6,8 +6,17 @@ import { flow } from "fp-ts/function"
  * @param str Raw JSON data to be parsed
  * @returns Option type with some(JSON data) or none
  */
-export const safeParseJSON = (str: string): O.Option<object> =>
-  O.tryCatch(() => JSON.parse(str))
+export const safeParseJSON = (
+  str: string,
+  convertToArray = false
+): O.Option<Record<string, unknown> | Array<unknown>> =>
+  O.tryCatch(() => {
+    const data = JSON.parse(str)
+    if (convertToArray) {
+      return Array.isArray(data) ? data : [data]
+    }
+    return data
+  })
```

Comment:
This function was used in a few places to parse JSON contents and return the results, thought to add an argument to determine the format of the return value. We could move this logic to where the hopp environment-specific parsing logic resides, but keeping it here might be helpful when a similar requirement arises for another import-based approach down the lane.